homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -8,29 +12,33 @@
 from homeassistant.helpers.entity import EntityCategory
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
-from .const import (
-    DOMAIN,
-    FLAME_ICON,
-    FLOW_OFF_ICON,
```

Comment:
Are the icon constants used somewhere else or can we remove them completely?

Reply:
They are used on some other places as well, once all platforms have been redone, they can all be removed.